recon/src/libp2p.rs

```diff
@@ -193,9 +203,14 @@ where
                         connections: vec![connection_info],
                         next_sync: BTreeMap::from_iter([
                             // Schedule all stream_sets initially
-                            (StreamSet::Interest, Instant::now()),
+                            (StreamSet::Peer, Instant::now()),
+                            // Schedule interests after peers
+                            (
+                                StreamSet::Interest,
+                                Instant::now() + Duration::from_millis(1),
```

Comment:
I went on a bit of squirrel to find out if instants are guaranteed to increase.. and indeed they these days barring platform bugs (some fun PRs and issues about it in rustc)